src/fluree/db/ledger/bootstrap.clj

```diff
@@ -98,16 +98,17 @@
 
 ;; TODO - too easy to forget to adjust this if we add a new collection type - we should
 ;; have an extra check when loading to ensure we have all the ecounts correct.
-(def genesis-ecount {const/$_predicate  (flake/->sid const/$_predicate 999)
+(def genesis-ecount {const/$_predicate  (flake/->sid const/$_predicate 1000)
                      const/$_collection (flake/->sid const/$_collection 19)
-                     const/$_tag        (flake/->sid const/$_tag 999)
-                     const/$_fn         (flake/->sid const/$_fn 999)
-                     const/$_user       (flake/->sid const/$_user 999)
-                     const/$_auth       (flake/->sid const/$_auth 999)
-                     const/$_role       (flake/->sid const/$_role 999)
-                     const/$_rule       (flake/->sid const/$_rule 999)
-                     const/$_setting    (flake/->sid const/$_setting 999)})
-                     ;const/$_shard      (flake/->sid const/$_shard 999)
+                     const/$_tag        (flake/->sid const/$_tag 1000)
+                     const/$_fn         (flake/->sid const/$_fn 1000)
+                     const/$_user       (flake/->sid const/$_user 1000)
+                     const/$_auth       (flake/->sid const/$_auth 1000)
+                     const/$_role       (flake/->sid const/$_role 1000)
+                     const/$_rule       (flake/->sid const/$_rule 1000)
+                     const/$_setting    (flake/->sid const/$_setting 1000)
+                     const/$_prefix     (flake/->sid const/$_prefix 1000)
+                     const/$_shard      (flake/->sid const/$_shard 1000)})
```

Comment:
Right now this is read-only and isn't being depended on. If it will be depended on we'll have to do a data upgrade, which can likely be handled with an extra block being added.